### PR TITLE
Origin/task/tec194/business update

### PR DESCRIPTION
**Pull request by geraldkachi**

### CODE REVIEW

File: src/app/register-business/page.tsx [hunk 1]
```diff
@@ -82,6 +85,7 @@ const RegisterBusiness = () => {
     : {};
   const tabsRef = useRef<HTMLDivElement>(null);
   const [activeTab, setActiveTab] = useState(0);
+  const [activeTabN, setActiveTabN] = useState(0);
```
Comment (Benrobo) [inline src/app/register-business/page.tsx]: any reason behind creating similar states with same functionalities?

File: src/app/register-business/page.tsx [hunk 2]
```diff
@@ -511,69 +520,156 @@ const RegisterBusiness = () => {
     return resp;
   };
 
+  const capitalizeWords = (str: string) => {
+    return str
+    .split('-') // Split the string by hyphens
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1)) // Capitalize the first letter of each word
+    .join(' '); // Join the array back into a string with spaces Join the array back into a string
+  };
+
```
Comment (Benrobo) [inline src/app/register-business/page.tsx:528]: Add this function within `utils/index.ts` file.

File: src/app/register-business/page.tsx [hunk 3]
```diff
@@ -511,69 +520,156 @@ const RegisterBusiness = () => {
...
-        )}
-        {activeTab === 1 && !successfulSubmission && (
-          <OperationInfo
-            formik={formik}
-            businessId={businessId}
-            isLoading={isLoading}
-          />
-        )}
-      </div>
+      {/* {(isDesktop && isUpdateMode) ? ( */}
```
Comment (Benrobo) [inline src/app/register-business/page.tsx]: remove redundant comment

**Review by Benrobo — CHANGES_REQUESTED** [review]
Also, this page currently had some well optimized updates so the changes you've added here won't reflect on dev, later on once we're done migrating all your UI changes, you would need to pull from dev to have a clean branch.

**Comment by geraldkachi** [comment]
Okay

**Pull request merged**